src/hypertable.c

```diff
@@ -836,6 +836,12 @@ table_has_replica_identity(Relation rel)
 	return rel->rd_rel->relreplident != REPLICA_IDENTITY_DEFAULT;
 }
 
+static bool
+table_has_rules(Relation rel)
+{
+	return rel->rd_rules != NULL;
```

Comment:
Maybe a simple NULL check like this doesn't warrant a full function, but instead fits better as a macro, or at least an inline function?